server/mongo_data_access.go

```diff
@@ -52,6 +100,11 @@ func (dal *mongoDataAccessLayer) PostWithID(id string, resource interface{}) err
 	resourceType := reflect.TypeOf(resource).Elem().Name()
 	collection := dal.Database.C(models.PluralizeLowerResourceName(resourceType))
 	updateLastUpdatedDate(resource)
+
+	if dal.hasInterceptorsForVerbAndType("POST", resourceType) {
```

Comment:
Since POST doesn't do anything extra before calling interceptors, this `if` statement seems unnecessary.  I think it's actually more efficient NOT to have it, because if there *are* interceptors, it requires the code to look through them _twice_ (once to check if there are any interceptors, and again when it invokes the interceptors).